style.css

```diff
@@ -0,0 +1,434 @@
+.container {
+  background: url(./img/background.jpg);
+  background-size: cover;
+  display: flex;
+  max-width: 1440px;
+  min-width: 320px;
+  height: 100vh;
+  margin: auto;
+}
+
+.menu { margin: 0 25px 0 100px; }
+
+.title {
+  color: #fff;
+  margin: 50px 0 20px 0;
+  font: normal bold 57px/ 67px 'Roboto', sans-serif;
+}
+
+.subtitle {
+  color: #fff;
+  margin: 0 0 60px 0;
+	font: normal bold 19px/ 21px 'Roboto', sans-serif;
+}
+
+.menu__level-selection {
+  position: relative;
+  margin-bottom: 70px;
+}
+
+.level_btn {
+  display: flex;
+  margin-bottom: 20px;
+  position: relative;
+  background: none;
+  border: none;
+  color: #fff;
+	font: normal bold 50px/ 59px 'Roboto', sans-serif;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/finaljs/pullRequest?prid=5949810)